csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.shared.cs

```diff
@@ -2093,6 +2121,20 @@ out IntPtr lora_adapter
 
         public static DOrtGetTensorMemoryInfo OrtGetTensorMemoryInfo;
 
+        [UnmanagedFunctionPointer(CallingConvention.Winapi)]
+        public delegate IntPtr /*(OrtStatus*)*/ DOrtGetExecutionProviderApi(byte[] /*(const char*)*/ provider_name,
+                                                                                uint /*(uint32_t)*/ version,
+                                                                                out IntPtr /* const OrtMemoryInfo** */ provider_api);
+
```

Comment:
Generally we require (although this was not enforced in the past) lines to be of 120 chars or less.
And, yes, we use default C# formating.